gordo_components/server/views/anomaly.py

```diff
@@ -93,6 +82,8 @@ class AnomalyView(BaseModelView):
      'time-seconds': '0.1937'}
     """
 
+    methods = ["POST"]
```

Comment:
I cant see where this variable is used. If it is a constant maybe it should BE CAPS!?!?!

Reply:
Flask uses it internally in their [Views](https://flask.palletsprojects.com/en/1.1.x/api/#flask.views.View) to auto return [405](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405)s